pkg/plugins/resources/cargopackage/condition.go

```diff
@@ -0,0 +1,62 @@
+package cargopackage
+
+import (
+	"errors"
+	"fmt"
```

Comment:
```suggestion
```
I don't think we need fmt in this file if we convert ftm.Printf to logrus.Printf